js/postView.js

```diff
@@ -1,88 +1,94 @@
 'use strict';
 
-var postView = {};
+(function(module) {
 
-postView.hamburgerMenu = function() {
-  $('.main-nav div').on('click', function() {
-    $(this).toggleClass('icon-cross');
-    $('.main-nav ul').toggle();
-  });
-};
+  var postView = {};
 
-postView.handleMainNav = function() {
-  $('.main-nav').on('click', '.tab', function(event) {
-    var $selectedTab = $(this).attr('data-content');
-    var $hamburger = $('.main-nav div');
-    $('section.tab-content').hide();
-    $('section.tab-content[id="' + $selectedTab + '"]').fadeIn('slow');
-    if($hamburger.hasClass('icon-cross')) {
-      $hamburger.toggleClass('icon-cross');
-      $(this).parent().fadeOut();
-    }
-  });
-  $('.main-nav .tab:last').click();
-};
+  postView.hamburgerMenu = function() {
+    $('.main-nav div').on('click', function() {
+      $(this).toggleClass('icon-cross');
+      $('.main-nav ul').toggle();
+    });
+  };
```

Comment:
Tried this, the hb menu doesn't work properly if it's inside the handle main nav function.